services/streammanager/kafka/kafkamanager.go

```diff
@@ -130,27 +130,31 @@ func NewProducer(destinationConfig interface{}, o Opts) (sarama.SyncProducer, er
 
 	var destConfig = Config{}
 	jsonConfig, err := json.Marshal(destinationConfig)
+	hosts := make([]string, 0)
 	if err != nil {
 		return nil, fmt.Errorf("[Kafka] Error while marshaling destination Config %+v, with Error : %w", destinationConfig, err)
 	}
 	err = json.Unmarshal(jsonConfig, &destConfig)
 	if err != nil {
 		return nil, fmt.Errorf("[Kafka] Error while unmarshalling dest config :: %w", err)
 	}
-	hostName := destConfig.HostName + ":" + destConfig.Port
+	hostNames := strings.Split(destConfig.HostName, ",")
+	for _, hostName := range hostNames {
+		hosts = append(hosts, hostName+":"+destConfig.Port)
+	}
 	isSslEnabled := destConfig.SslEnabled
 
-	hosts := []string{hostName}
-
 	config := getDefaultConfiguration()
 	config.Producer.Timeout = o.Timeout
 
 	if isSslEnabled {
 		caCertificate := destConfig.CACertificate
-		tlsConfig := NewTLSConfig(caCertificate)
-		if tlsConfig != nil {
-			config.Net.TLS.Config = tlsConfig
-			config.Net.TLS.Enable = true
+		config.Net.TLS.Enable = true
+		if caCertificate != "" {
```

Comment:
Wouldn't it be safer to have an explicit `destConfig.CustomCertificate` bool option instead of checking for empty string ?

Reply:
So you mean to say we should have another toggle on UI to use customer certificate?